cves/kernel/CVE-2022-3108.yml

```diff
@@ -448,7 +452,7 @@ mistakes:
 
```

Comment:
Where is the common lessons learned? There has to be at least one unless i'm tripping and cannot find it.

Reply:
Ok Fixed that